.github/release-drafter.yml

```diff
@@ -0,0 +1,2 @@
+_extends: .github
+tag-template: jetty-xhtml-schemas-$NEXT_MINOR_VERSION
```

Comment:
we won't use releases on this repo.

Reply:
why not?
it's good to see the changes
it's cheap to use

Reply:
Again, there's nobody but us that use it as a dependency.
Doing format github releases for the purposes of release notes on dependabots is pointless here.
Remove this configuration from this PR.